algorithms/kernel/k_nearest_neighbors/kdtree_knn_classification_predict_dense_default_batch_impl.i

```diff
@@ -348,7 +348,6 @@ void KNNClassificationPredictKernel<algorithmFpType, defaultDense, cpu>::findNea
 
     DAAL_ALIGNAS(256) algorithmFpType distance[__KDTREE_LEAF_BUCKET_SIZE + 1];
     size_t start, end;
-    algorithmFpType dist, diff, val;
```

Comment:
Double check that you do not affect performance here

Reply:
Ok